Makefile

```diff
@@ -6,10 +6,10 @@
 TOP = .
 include $(TOP)/build.mk
 
-SUB_DIRS = common utils ercc ecc_enclave ecc tlcc_enclave tlcc plugins fabric examples integration demo # docs
+SUB_DIRS = common utils ercc ecc_enclave ecc tlcc_enclave tlcc fabric examples integration demo # docs
 
 FPC_SDK_DEP_DIRS = utils/fabric common ecc_enclave ecc
-FPC_PEER_DEP_DIRS = plugins tlcc_enclave tlcc fabric ecc_enclave ecc # ecc & ecc_enclave we need if we run on host in external builder, not via docker
+FPC_PEER_DEP_DIRS = ercc tlcc_enclave tlcc fabric ecc_enclave ecc # ecc & ecc_enclave we need if we run on host in external builder, not via docker
```

Comment:
the validation does not have any build artefacts anymore as the code is used when build in the peer in `fabric/`. In contrast, in `plugins/` we compiled the actual plugins, so I would say from regarding build structure they are now equivalent. 

However, since the ercc-vscc needs the `provider.go` in `ercc/attestation/`, we need to compile this first, before running `make` in `fabric/` to build the peer.

Reply:
With ercc refactoring and removing ercc-vscc this build dep will go away.

Reply:
key phrase, though, "wll", i.e., "did not yet" ;-)